packaging/helm-charts/helm3/strimzi-kafka-operator/crds/041-Crd-kafkaconnect.yaml

```diff
@@ -1727,27 +1727,39 @@ spec:
                                 fileName:
                                   type: string
                                   description: Name under which the artifact will be stored.
+                                group:
+                                  type: string
+                                  description: Group of the artifact. Applicable for 'maven' type of artifact only.
+                                repository:
+                                  type: string
+                                  description: The repository to download the artifact from. Applicable for 'maven' type of artifact only.
                                 sha512sum:
                                   type: string
-                                  description: SHA512 checksum of the artifact. Optional. If specified, the checksum will be verified while building the new container. If not specified, the downloaded artifact will not be verified.
+                                  description: SHA512 checksum of the artifact. Optional. If specified, the checksum will be verified while building the new container. If not specified, the downloaded artifact will not be verified. Not applicable for 'maven' type of artifact.
                                 type:
                                   type: string
                                   enum:
                                     - jar
                                     - tgz
                                     - zip
+                                    - maven
                                     - other
-                                  description: Artifact type. Currently, the supported artifact types are `tgz`, `jar`, and `zip`.
+                                  description: Artifact type. Currently, the supported artifact types are `tgz`, `jar`, `zip` and `maven`.
                                 url:
                                   type: string
                                   pattern: ^(https?|ftp)://[-a-zA-Z0-9+&@#/%?=~_|!:,.;]*[-a-zA-Z0-9+&@#/%=~_|]
-                                  description: URL of the artifact which will be downloaded. Strimzi does not do any security scanning of the downloaded artifacts. For security reasons, you should first verify the artifacts manually and configure the checksum verification to make sure the same artifact is used in the automated build. Required.
+                                  description: URL of the artifact which will be downloaded. Strimzi does not do any security scanning of the downloaded artifacts. For security reasons, you should first verify the artifacts manually and configure the checksum verification to make sure the same artifact is used in the automated build. Required. Not applicable for 'maven' type of artifact.
+                                version:
+                                  type: string
+                                  description: Version of the artifact. Applicable for 'maven' type of artifact only.
```

Comment:
```suggestion
                                  description: Version of the artifact. Applicable to the `maven` artifact type only.
```

Reply:
We are specifying artifact, not maven version. I think this is ambiguous.